regtests/t_spark_sql/src/spark_sql_s3_cross_region.sh

```diff
@@ -24,7 +24,7 @@ if [ -z "$AWS_CROSS_REGION_TEST_ENABLED" ] || [ "$AWS_CROSS_REGION_TEST_ENABLED"
   exit 0
 fi
 
-SPARK_BEARER_TOKEN="${REGTEST_ROOT_BEARER_TOKEN:-principal:root;realm:realm1}"
+SPARK_BEARER_TOKEN="${REGTEST_ROOT_BEARER_TOKEN:-principal:root;realm:POLARIS}"
```

Comment:
Do we need to keep `-principal:root;realm:POLARIS` as token will be fetched by `run.sh`? Given the direction that we want to to is to use the default authenticator always for reggresstion test, I think we should remove them to avoid confusion.

Reply:
With @HonahX we decided to postpone that, but I think you are right: leaving this default value will create confusion. So I went ahead and removed them.